js/crm.menubar.js

```diff
@@ -411,7 +411,7 @@
     branchTpl:
       '<% _.forEach(items, function(item) { %>' +
         '<li <%= attr("li", item) %>>' +
-          '<a <%= attr("a", item) %>>' +
+          '<a <%= attr("a", item) %> <% if (item.target) { %>target=<%- item.target %><% } %>>' +
```

Comment:
Looking at the `attr()` function, maybe instead of patching this line you could do this:

```diff
diff --git a/js/crm.menubar.js b/js/crm.menubar.js
index 152a6e376e..9fa49f63d6 100644
--- a/js/crm.menubar.js
+++ b/js/crm.menubar.js
@@ -471,7 +471,7 @@
   }
 
   function attr(el, item) {
-    var ret = [], attr = _.cloneDeep(item.attr || {}), a = ['rel', 'accesskey'];
+    var ret = [], attr = _.cloneDeep(item.attr || {}), a = ['rel', 'accesskey', 'target'];
     if (el === 'a') {
       attr = _.pick(attr, a);
       attr.href = item.url || "#";
```

Reply:
@monishdeb any thoughts?